network-mux/src/Network/Mux/Types.hs

```diff
@@ -80,14 +80,15 @@ newtype MiniProtocolNum = MiniProtocolNum Word16
   deriving (Eq, Ord, Enum, Ix, Show)
 
 -- | Per Miniprotocol limits
-data MiniProtocolLimits =
+newtype MiniProtocolLimits =
      MiniProtocolLimits {
        -- | Limit on the maximum number of bytes that can be queued in the
        -- miniprotocol's ingress queue.
        --
-       maximumIngressQueue :: !Int
+       maximumIngressQueue :: Int
      }
-
+  -- GR-FIXME: Q. if part of wire format, shouldn't this be a fixed width, E.g.,
+  -- Word16?
```

Comment:
It is not part of the [wire format](https://github.com/input-output-hk/ouroboros-network/blob/master/network-mux/src/Network/Mux/Codec.hs).  The value depends on how large are messages and how much pipelining a mini-protocol does.